examples/sycl/pvc/pvc_collective_builder.cpp

```diff
@@ -200,13 +190,9 @@ struct ExampleRunner {
 
     syclcompat::wait();
 
-    // Check if output from CUTLASS kernel and reference kernel are relatively equal or not
-    auto epsilon = static_cast<ElementOutput>(0.1f);
-    auto nonzero_floor = static_cast<ElementOutput>(0.1f);
-
-    bool passed = cutlass::reference::device::BlockCompareRelativelyEqual(
-            block_ref_D.get(), block_D.get(), block_D.size(),
-            epsilon, nonzero_floor);
+    // Check if output from CUTLASS kernel and reference kernel are equal or not
+    bool passed = cutlass::reference::device::BlockCompareEqual(
```

Comment:
[Question] why are we using an exact comparison for floating point compute, and from what I can see, BlockFillRandomUniform does not initialize with exact zeros either ?

Reply:
Float types are cast to integers to facilitate testing. See [here](https://github.com/codeplaysoftware/cutlass-fork/blob/64acac8f1043375d11dfbe93dca04a841f68e773/tools/util/include/cutlass/util/reference/device/tensor_fill.h#L485) for the cuda implementation and [here](https://github.com/codeplaysoftware/cutlass-fork/blob/64acac8f1043375d11dfbe93dca04a841f68e773/tools/util/include/cutlass/util/reference/device/sycl_tensor_fill.h#L132) for the sycl equivalent.